ui/src/Drawer.tsx

```diff
@@ -9,23 +10,67 @@ import SearchResults from "./SearchResults";
 import { ScholarReaderContext } from "./state";
 
 const PDF_VIEWER_DRAWER_OPEN_CLASS = "drawer-open";
-
+const BLACK_LISTED_CLASS_NAME = "MuiTooltip-tooltip";
 export class Drawer extends React.PureComponent {
+  static contextType = ScholarReaderContext;
+  context!: React.ContextType<typeof ScholarReaderContext>;
+
   componentWillUnmount() {
     const { pdfViewer } = this.context;
-    if (pdfViewer !== undefined && pdfViewer !== null) {
+    if (pdfViewer != null) {
```

Comment:
Why was the check for `undefined` removed?

Reply:
!= null is equivalent to !== undefined && !== null due to JS type coercion for single equals. It was used at many of my previous internships so I thought I would introduce it here hahah.

Reply:
Roger! Okay, this is fine.